README.md

```diff
@@ -5,12 +5,22 @@ fully featured Offline Workflow and Conflict Resolution.
 
 ## Features 
 
- 
 - Offline support. Mutations are persisted when Offline.
 - Offline Listeners and workflows for seamless UI.
 - Flexible, out of the box Conflict Resolution implementations
 - Subscriptions and Binary Upload that works offline.
 
+## Offline Support is implemented where the mutations are persisted when the client is offline.
+
+#### Steps Followed:
+A database is maintained which stores the class name of the mutation and the value map of variables given by user along with some other details like operationID, etc.
+
+- When the client comes online, clicking on the "FROMDB" button will take mutation from DB by iterating over a list of mutations stored in the offline queue.
```

Comment:
Kinda strange to have this info in README as we reference to sample app.
Maybe sample app readme will be better?